Sitefinity CLI/Services/ProjectService.cs

```diff
@@ -0,0 +1,164 @@
+﻿using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using Sitefinity_CLI.Exceptions;
+using Sitefinity_CLI.Services.Interfaces;
+using Sitefinity_CLI.VisualStudio;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Sitefinity_CLI.Model;
+using Microsoft.Extensions.Logging;
+
+namespace Sitefinity_CLI.Services
+{
+    internal class ProjectService : IProjectService
```

Comment:
Project service is very generic naming. Also the methods that are included are not related to "Projects"
GetLatestSitefinityVersion()  GetConfigsForProjectsWithoutSitefinity() RestoreReferences,

Think of a better naming. Maybe extract different services or add the methods in some of the existing ones